content/01.abstract.md

```diff
@@ -1,12 +1,25 @@
 ## Abstract {.page_break_before}
 
-Knowledge graphs support multiple research efforts by providing contextual information for biomedical entities, constructing networks, and supporting the interpretation of high-throughput analyses. 
-These databases are populated via some form of manual curation, which is challenging to scale with an increasing publication rate. 
+### Background
+
+Knowledge graphs support biomedical research efforts by providing contextual information for biomedical entities, constructing networks, and supporting the interpretation of high-throughput analyses. 
+These databases are populated via some form of manual curation, which is challenging to scale with an exponentially rising publication rate. 
 Data programming is a paradigm that circumvents this arduous manual process by combining databases with simple rules and heuristics written as label functions, which are programs designed to annotate textual data automatically. 
 Unfortunately, writing a useful label function requires substantial error analysis and is a nontrivial task that takes multiple days per function.
-This circumstance makes populating a knowledge graph with multiple nodes and edge types practically infeasible. 
+This bottleneck makes populating a knowledge graph with multiple nodes and edge types practically infeasible.
 We sought to accelerate the label function creation process by evaluating how label functions can be re-used across multiple edge types.
-We used a subset of an existing knowledge graph centered on disease, compound, and gene entities to evaluate label function re-use. 
-We determined the best label function combination by comparing a baseline database-only model with the same model but added edge-specific or edge-mismatch label functions.
-We confirmed that adding additional edge-specific rather than edge-mismatch label functions often improves text annotation and shows that this approach can incorporate novel edges into our source knowledge graph. 
-We expect that the continued development of this strategy has the potential to swiftly populate knowledge graphs with new discoveries, ensuring that these resources include cutting-edge results.
+We obtained entity-tagged abstracts from Pubtator Central and subsetted these entities to only contain compounds, genes, and diseases mentions.
+Using an existing knowledge graph, Hetionet v1, we extracted sentences that mentioned the following biomedical relationships: Compound treats Disease (CtD), Gene interacts Gene (GiG), Compound binds Gene (CbG) and Disease associates Gene (DaG).
+We trained a baseline model that used database-only label functions and then used a sampling approach to measure how well adding edge-specific or edge-mismatch label function combinations improve over our baseline.
```

Comment:
These two sentences feel like results. Can you move this + maybe the two sentences above that are a bit concrete about what you did to results and we can take another look?